src/views/wedo2/wedo2.jsx

```diff
@@ -44,6 +44,14 @@ var Wedo2 = React.createClass({
                                 </h4>
                                 <p>
                                     <FormattedHTMLMessage id='wedo2.installText' />
+                                    <br />
+                                    <a href="https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=1084869222&mt=12">
+                                        <FormattedMessage id='wedo2.downloadMac' />
+                                    </a>
+                                    <br />
+                                    <a href="/">
```

Comment:
New link: https://downloads.scratch.mit.edu/device-manager/ScratchDeviceManager-1.1.0.exe
It's still not the final file but I think we can treat that as the final link.